apps/full-site-editing/full-site-editing-plugin/premium-content/index.js

```diff
@@ -33,11 +39,40 @@ const registerBlock = ( block ) => {
 	} );
 };
 
+/**
+ * Appends a "paid" tag to the Premium Content block title if site requires an upgrade.
+ */
+const addPaidBlockFlags = async () => {
+	const membershipsStatus = await apiFetch( { path: '/wpcom/v2/memberships/status' } );
+	const shouldUpgrade = membershipsStatus.should_upgrade_to_access_memberships;
+	if ( shouldUpgrade ) {
+		const premiumContentBlock = getBlockType( container.name );
+		if ( ! premiumContentBlock ) {
+			return;
+		}
+
+		const paidFlag = _x(
```

Comment:
I had to look up this one. Signature is _x( $text, $context, $domain ). The advantage over `__` is translator context and multiple translations per context.

https://developer.wordpress.org/reference/functions/_x/

Reply:
TBH I copied from what we do for the Jetpack block:

https://github.com/Automattic/jetpack/blob/796147bfa0fd7d15661ca41337a37f05ca19cf69/extensions/shared/register-jetpack-block.js#L17